src/lang/extra/galician.txt

```diff
@@ -737,6 +740,7 @@ STR_TRANSPARENT_TUNNELS_TOOLTIP                                 :{BLACK}Alterna
 
 STR_LINKGRAPH_STATS_TOOLTIP_TIME_EXTENSION_GENERAL              :{}Tempo de viaxe promedio: {STRING}
 STR_LINKGRAPH_STATS_TOOLTIP_CAPACITY                            :{}Capacidade: {CARGO_LONG}
+STR_LINKGRAPH_STATS_TOOLTIP_DISTANCE                            :Distancia Manhattan: {COMMA} cadro{P 0 "" s}{}Distancia en liñaBird Fly Distance: {COMMA} tile{P 1 "" s}
```

Comment:
```suggestion
STR_LINKGRAPH_STATS_TOOLTIP_DISTANCE                            :Distancia Manhattan: {COMMA} cadro{P 0 "" s}{}Distancia en liña: {COMMA} cadro{P 1 "" s}
```

Reply:
Thanks @James103 , I'll have to check a third time after the commit, this one went unnoticed under my nose!